styles/Home.module.css

```diff
@@ -1,51 +1,60 @@
-/*.container_1{
-    position: relative;
-}*/
-
-.line{
-    display: flex;
+.bg_color{
     background-color: #e7f5fe;
 }
 
-.column_50{
-    width: 50%;
-}
-
-.column_50 img{
-    width: auto;
-    height: auto;
-    position:relative;
-    left: 0px;
-    transform: translateX(-30%)
+/*First Container*/
+.image{
+    transform: translateX(-30%);
+    max-width: none;
 }
 
 .journal{
-    position: absolute;
-    top: 30%;
-    left: 55%;
-    margin: -25px 0 0 -25px;
-    max-width: 520px;
+    max-width: 550px;
+    margin: 0 auto;
 }
 
 .journal h1{
-    font-weight: bold;
     color: #03395d;
     letter-spacing: 5px;
-    font-size: 5em;
 }
 
 .journal p{
     color: #03395d;
-    font-size: 1.6em;
 }
 
 .btn_component{
-    display: flex;
     gap: 20px;
 }
 
+/*Media - first container*/
+@media(min-width: 960px){
```

Comment:
Se formos nesse caminho, é melhor extrair isso para um arquivo que é importado automaticamente em todos os cantos, senão vc vai ter que repetir essa lógica em mtos lugares.

Se for usar tachyons, é só fazer, por exemplo nos **h1**: `f4 f5-l` algo assim. O que isso significa: em qualquer dispositivos, use a escala 4, se o dispositivo for grande (`large`), use a escala 5.